### PR TITLE
Popup issues

### PR DESCRIPTION
**Pull request by TheCovenant**
I had to get rid of the changes I made to remove empty attributes off of the Popup for now.
My approach for getting them off was to create a new Feature interface that made all attributes that could be null, optional. This made it so essentially there were no empty fields/attributes in a feature because if an attribute would be null it would just be excluded from the feature altogether.

Unfortunately, I could not keep this change in my PR as it was breaking my filters since right now they depend on all features having all attributes regardless of whether the attribute is null or not.

### CODE REVIEW

**Comment by gordom6** [comment]
The empty attributes are fine. Changing the Dataset name and removing __typename was more important.

**Comment by gordom6** [comment]
Still need to remove __typename?

**Comment by gordom6** [comment]
I seem to be unable to get rid of this popup:
Screen Shot 2020-05-11 at 9 50 33 AM

**Comment by gordom6** [comment]
Don't need or care about uri attribute.

File: app/geo/gui/src/ts/twxplore/gui/geo/featureTypeStrategies/PointFeatureTypeStrategy/PointFeatureTypeStrategy.ts [hunk 1]
```diff
@@ -1,7 +1,13 @@
 import {FeatureTypeStrategy} from "../FeatureTypeStrategy";
-import {FeatureType} from "../../api/graphqlGlobalTypes";
+import {FeatureType, FeatureQuery} from "../../api/graphqlGlobalTypes";
 
 export abstract class PointFeatureTypeStrategy implements FeatureTypeStrategy {
+  getQuery(clickedFeatureUri: string): FeatureQuery {
```
Comment (gordom6) [inline app/geo/gui/src/ts/twxplore/gui/geo/featureTypeStrategies/PointFeatureTypeStrategy/PointFeatureTypeStrategy.ts:5]: This doesn't make much sense. A point can never have a within query. Return null to indicate no query should be issued?

File: app/geo/gui/src/ts/twxplore/gui/geo/featureTypeStrategies/RootFeatureTypeStrategy.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import {FeatureTypeStrategy} from "./FeatureTypeStrategy";
+import {FeatureType, FeatureQuery} from "../api/graphqlGlobalTypes";
+
+export class RootFeatureTypeStrategy implements FeatureTypeStrategy {
+  getQuery(clickedFeatureUri: string): FeatureQuery {
+    return {
+      onlyFeatureUri:
+        "http://twxplore.github.io/app/geo/feature#tiger_line-tl_2019_us_state-38",
```
Comment (gordom6) [inline app/geo/gui/src/ts/twxplore/gui/geo/featureTypeStrategies/RootFeatureTypeStrategy.ts:8]: Put this magic string in a constant, please.
NEW_YORK_STATE_FEATURE_URI

File: app/geo/gui/src/ts/twxplore/gui/geo/states/map/initialMapState.ts [hunk 3]
```diff
@@ -39,11 +42,25 @@ Object.values(FeatureType).map(featureType => {
   });
 });
 
-//const featureTypesFilters: {[featureType: string]: MapFilterState} = {};
-
+const features = [];
+const rootFeature: MapFeature = {
+  __typename: "Feature",
+  regions: [],
+  uri: "root-uri",
```
Comment (gordom6) [inline app/geo/gui/src/ts/twxplore/gui/geo/states/map/initialMapState.ts:49]: http://twxplore.github.io/app/geo/feature#root

**Comment by TheCovenant** [comment]
Policy features won't show on the map yet, as I haven't yet added them to withinFeatureTypes of any of my FeatureTypeStrategy subclasses. 
That should be for another branch I think.

File: app/geo/gui/src/ts/twxplore/gui/geo/states/map/ROOT_FEATURE_URI.ts [hunk 4]
```diff
@@ -0,0 +1,3 @@
+type ROOT_FEATURE_URI = "http://twxplore.github.io/app/geo/feature#root";
+export const ROOT_FEATURE_URI: ROOT_FEATURE_URI =
```
Comment (gordom6) [inline app/geo/gui/src/ts/twxplore/gui/geo/states/map/ROOT_FEATURE_URI.ts:2]: You don't really need the type, that's mostly for action names where you want to do a switch.

**Pull request merged**